src/main/java/com/tstepnik/planner/controller/UserController.java

```diff
@@ -17,32 +19,37 @@
 public class UserController {
 
     private final UserService userService;
+    private final UserMapper mapper;
 
-    public UserController(UserService userService) {
+    public UserController(UserService userService, UserMapper mapper) {
         this.userService = userService;
+        this.mapper = mapper;
     }
 
     @GetMapping
     @PreAuthorize("hasRole('ROLE_ADMIN')")
-    public ResponseEntity<List<User>> getUsers() {
-        return ResponseEntity.ok(userService.getUsers());
+    public ResponseEntity<List<UserDto>> getUsers() {
+        List<User> users = userService.getUsers();
+        return ResponseEntity.ok(mapper.toDto(users));
+
     }
 
     @GetMapping("/me")
     @PreAuthorize("hasRole('USER')")
-    public ResponseEntity<User> getLoggedUser(Principal principal) {
+    public ResponseEntity<UserDto> getLoggedUser(Principal principal) {
         CustomUserDetails customUserDetails = (CustomUserDetails) ((UsernamePasswordAuthenticationToken) principal).getPrincipal();
-        return ResponseEntity.ok(customUserDetails.getUser());
+        User user = customUserDetails.getUser();
+        return ResponseEntity.ok(mapper.toDto(user));
     }
 
     @PutMapping("/{id}")
     @PreAuthorize("hasRole('USER')")
-    public ResponseEntity<User> updatedUser(@RequestBody User user, @PathVariable("id") long id) {
-        Optional<User> updateUser = Optional.ofNullable(userService.updateUser(user, id));
+    public ResponseEntity<UserDto> updatedUser(@RequestBody UserDto user, @PathVariable("id") long id) {
+        Optional<User> updateUser = Optional.ofNullable(userService.updateUser(mapper.toUser(user), id));
         if (updateUser.isEmpty()) {
-            return new ResponseEntity<User>(updateUser.get(), HttpStatus.CONFLICT);
+            return new ResponseEntity<UserDto>(mapper.toDto(updateUser.get()), HttpStatus.CONFLICT);
```

Comment:
//todo remove once global exception handler is added ?

Reply:
I added now, it wos there but probably thorugh changes in autorization someone removed it.